tests/test_Guards.py

```diff
@@ -35,7 +36,114 @@ def test_Guards__safe_builtins__2(e_exec):
         __name__='restricted_module',
         __metaclass__=type,
         _write_=lambda x: x,
-        _getattr_=getattr)
+        _getattr_=getattr,)
 
     e_exec(CLASS_SOURCE, restricted_globals)
     assert restricted_globals['b'] == '2411'
+
+
+class D:
```

Comment:
The classes should have more meaningful values. If the are only needed for one test they could be moved into the test function together with the code which uses them.